src/test/testOrderBook.java

```diff
@@ -51,5 +69,77 @@ void placeOneBid() {
 		assertEquals(-1f, testSubject.bestAskPrice); // Should be unaffected
 		assertEquals(3.39f, testSubject.bestBidPrice);
 		assertEquals(0.25f, testSubject.sittingVolume);
+
+		// Init and checking start conditions
+		OrderBook testSubject2 = new OrderBook();
+
+		// Operation
+		Order newBid2 = new Order(6.519f, 0.75f, Side.BID, 11110011L);
+		testSubject2.placeOrder(newBid2);
+		// Checking Operation
+		assertEquals(0, testSubject2.asks.size());
+		assertEquals(1, testSubject2.bids.size());
+		assertEquals(6.519f, testSubject2.bids.getFirst().price);
+		assertEquals(0.75f, testSubject2.bids.getFirst().size);
+		assertEquals(11110011L, testSubject2.bids.getFirst().createdTs);
+		assertEquals(-1f, testSubject2.bestAskPrice); // Should be unaffected
+		assertEquals(6.519f, testSubject2.bestBidPrice);
+		assertEquals(0.75f, testSubject2.sittingVolume);
 	}
+
+	@Test
+	void placeMultipleOrders() {
+		// Init
+		OrderBook testSubject = new OrderBook();
+
+		// Operation
+		Order newBid = new Order("123","bid1", 4.39f, 0.25f, OrderType.MARKET_ORDER, true,
+				Side.BID, 1677541289L);
+		testSubject.placeOrder(newBid);
+
+		Order newBid2 = new Order("23","bid2", 5.01f, 0.3f, OrderType.MARKET_ORDER, true,
+				Side.BID, 1677541212L);
+		testSubject.placeOrder(newBid2);
+
+		Order newAsk = new Order("32","ask1", 2.25f, 0.4f, OrderType.MARKET_ORDER, true,
+				Side.ASK, 167754132L);
+		testSubject.placeOrder(newAsk);
+
+		Order newAsk2 = new Order("3","ask2", 1.25f, 0.35f, OrderType.MARKET_ORDER, true,
+				Side.ASK, 167754132L);
+		testSubject.placeOrder(newAsk2);
+
+		// Checking Operation
+		assertEquals(2, testSubject.bids.size());
+		assertEquals(2, testSubject.asks.size());
+		assertEquals(0.3f, testSubject.bids.getFirst().price);
+		assertEquals(0.4f, testSubject.bestAskPrice);
+		assertEquals(0.3f, testSubject.bestBidPrice);
+		assertEquals(12.9f, testSubject.sittingVolume);
+	}
+
+	@Test
+	void OneFillEvent() {
+		// Init
+		OrderBook testSubject = new OrderBook();
+
```

Comment:
Add the filler code from above so we know we start of with nothing in the orderbook, then after you add your order we have something in the order book and then the fill reduces it back to 0. It could be the case that our create order and fill both don't work so don't even touch the actual orderbook